tests/e2e/tests/devfiles/JavaVertx.spec.ts

```diff
@@ -8,145 +8,45 @@
-        test('Wait IDE availability', async () => {
-            await ide.waitWorkspaceAndIde(namespace, workspaceName);
-        });
-
-        test('Open project tree container', async () => {
-            await projectTree.openProjectTreeContainer();
-        });
-
-        test('Wait project imported', async () => {
-            await projectTree.waitProjectImported(sampleName, 'src');
-        });
 
+    suite (`Create ${stack} workspace ${workspaceName}`, async () => {
+        workspaceHandling.createAndOpenWorkspace(workspaceName, stack);
+        projectAndFileTests.waitWorkspaceReadiness(workspaceName, sampleName, 'src');
     });
 
     suite('Language server validation', async () => {
-        test('Expand project and open file in editor', async () => {
-            await projectTree.expandPathAndOpenFileInAssociatedWorkspace(fileFolderPath, tabTitle);
-            await editor.selectTab(tabTitle);
-        });
-
-        test('Java LS initialization', async () => {
-            await ide.checkLsInitializationStart('Starting Java Language Server');
-            await ide.waitStatusBarTextAbsence('Starting Java Language Server', 1800000);
-            await ide.waitStatusBarTextAbsence('Building workspace', 360000);
-        });
-
-        test('Suggestion invoking', async () => {
-            await ide.closeAllNotifications();
-            await editor.waitEditorAvailable(tabTitle);
-            await editor.clickOnTab(tabTitle);
-            await editor.waitEditorAvailable(tabTitle);
-            await editor.waitTabFocused(tabTitle);
-            await editor.moveCursorToLineAndChar(tabTitle, 19, 31);
-            await editor.pressControlSpaceCombination(tabTitle);
-            await editor.waitSuggestionWithScrolling(tabTitle, 'router(Vertx vertx) : Router');
-        });
-
-        test('Error highlighting', async () => {
-            await editor.type(tabTitle, 'error', 20);
-            await editor.waitErrorInLine(21);
-            await editor.performKeyCombination(tabTitle, Key.chord(Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE));
-            await editor.waitErrorInLineDisappearance(21);
-        });
-
-        test('Autocomplete', async () => {
-            await editor.moveCursorToLineAndChar(tabTitle, 19, 7);
-            await editor.pressControlSpaceCombination(tabTitle);
-            await editor.waitSuggestionContainer();
-            await editor.waitSuggestionWithScrolling(tabTitle, 'Router - io.vertx.ext.web');
-        });
-
-        test('Codenavigation', async () => {
-            await editor.moveCursorToLineAndChar(tabTitle, 19, 7);
-            await editor.performKeyCombination(tabTitle, Key.chord(Key.CONTROL, Key.F12));
-            await editor.waitEditorAvailable(codeNavigationClassName);
-        });
-
+        projectAndFileTests.openFile(fileFolderPath, tabTitle);
+        commonLSTests.waitLSInitialization(LSstarting, 1800000, 360000);
```

Comment:
it would be great to name variables the same names because it is a bit confusing.
I mean ```LSTests``` and ```commonLSTests```.
I vote for ```commonLsTests```